tests/ModelTest.php

```diff
@@ -834,12 +834,12 @@ public function testChunkById(): void
         User::create(['name' => 'spork', 'tags' => ['sharp', 'pointy', 'round', 'bowl']]);
         User::create(['name' => 'spoon', 'tags' => ['round', 'bowl']]);
 
-        $count = 0;
-        User::chunkById(2, function (EloquentCollection $items) use (&$count) {
-            $count += count($items);
+        $names = [];
+        User::chunkById(2, function (EloquentCollection $items) use (&$names) {
+            $names = array_merge($names, $items->pluck('name')->all());
```

Comment:
I noted your response in https://github.com/GromNaN/laravel-mongodb/pull/25#discussion_r1300689945.

Not sure why I thought `+` concatenated instead of overwrote for numeric indexes. I certainly confused it with `array_merge()`, which is what we do need to use here :+1: